docs/explanations/pro_token_and_machine_usage.rst

```diff
@@ -0,0 +1,65 @@
+.. _pro_token_and_machine_usage:
+
+Machine counts and Token usage for Ubuntu Pro
+*********************************************
+
+Pro Tokens:
+-----------
+
+- Obtaining Tokens:
+    After subscribing to Ubuntu Pro, you can access your tokens through the Ubuntu Pro dashboard. Each token can be used to attach multiple machines depending on the subscription.
+
+- Attaching Tokens:
+    Command: Use ``sudo pro attach <token>`` to connect a machine to your Ubuntu Pro subscription.
+
+- Detaching Tokens:
+    Command: Use ``sudo pro detach`` to remove a machine from the subscription.
+
+
+Monitoring Active Machines:
+---------------------------
+
+- Machine Limits: 
+    Each subscription type has a limit on the number of machines it can cover. For personal subscriptions, this limit is 5 physical machines with unlimited VMs on these machines. For enterprise subscriptions, the limit is based on the number of physical hosts and VMs.
+
+- Active Machine Tracking: 
+    The Ubuntu Pro dashboard and tools like Landscape help track the number of active machines, ensuring compliance with subscription limits.
+
+
+- Unused Licenses and Active Users:
+    The Ubuntu Pro dashboard provides a comprehensive view of all active and attached machines. It shows the number of active machines and helps track unused licenses, ensuring that you remain within the subscription limits.
```

Comment:
Probably should link to the part of the dashboard that shows this info

Reply:
I think licenses is a Landscape-specific thing? So we might want to specify "Landscape licenses"